exercises/primzahlen.py

```diff
@@ -1,19 +1,19 @@
-primzahlen = []
-rechnen = [2, 3, 5, 7]
-for i in range(1, 100):
-    if i > 1:
-        for x in range(3):
-            if (i % rechnen[x] == 0):
-                break
-        else:
-            primzahlen.append(i)
-print(primzahlen)
+#primzahlen = []
+#rechnen = [2, 3, 5, 7]
+# for i in range(1, 100):
+#   if i > 1:
+#       for x in range(3):
+#           if (i % rechnen[x] == 0):
+#                break
+#       else:
+#           primzahlen.append(i)
+# print(primzahlen)
 
 primzahlen = []
 for i in range(1, 100):
     if i > 1:
         for x in range(2, i):
-            if (i % x == 0):
+            if (i % x) == 0:
```

Comment:
Die Klammern dürfen Sie machen, wenn Sie der Klarheit helfen (hier ok). Sie sind in diesem Beispiel aber prinzipiell gar nicht nötig :)
```suggestion
            if i % x == 0:
```